src/core/regimes.rkt

```diff
@@ -173,64 +173,80 @@
      (and (> pidx 0)) (list-ref can-split? pidx))
    (= (si-pidx (last split-indices)) (length can-split?))))
 
-(define/contract (err-lsts->split-indices err-lsts can-split-lst)
-  (->i ([e (listof list)] [cs (listof boolean?)]) [result (cs) (curry valid-splitindices? cs)])
+(define (err-lsts->split-indices err-lsts-vec can-split-vec)
```

Comment:
So here the `err-lsts` are a list (per alt) of lists (per point) of errors. I don't think it's important to have the outer list be a vector but the inner lists should be. This PR is kind of backwards there.